static/json/languages/en_US.json

```diff
@@ -1351,7 +1351,9 @@
       "view_gallery": "View Gallery"
     },
     "toast": {
-      "spam_reported": "Your report has been received!"
+      "spam_reported": "Your report has been received!",
+      "hidden": "NFT hidden",
+      "unhidden": "NFT unhidden"
```

Comment:
Lmk if there are any better names we can go for here. Currently some NFT names are very long so wanted to simplify it.